workflow/apm/span-list.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: APM Dashboard - Span List
+sidebarTitle: Span List
+---
+
+The APM Span List Dashboard displays all collected spans across your instrumented APMs in a centralized dashboard. This page provides you with detailed visibility into the execution flow of individual operations within a transaction or request.
+
+<Note> For more information on drilling into your spans, navigate to [Drilling Into Your Data](). </Note>
```

Comment:
We should take some time to explain the drill on this page as well, and then link over the drilling into your data for more info. Nothing here doesn't help much. 

Maybe this page was unfinished? Feels like it needs a lot more on how the flame chart works, the span list, etc.